erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -68,3 +73,48 @@ def get_expense_approver(doctype, txt, searchfield, start, page_len, filters):
 		where u.name = r.parent and r.role = 'Expense Approver' 
 		and u.enabled = 1 and u.name like %s
 	""", ("%" + txt + "%"))
+
+@frappe.whitelist()
+def make_bank_entry(docname):
+	from erpnext.accounts.doctype.journal_entry.journal_entry import get_default_bank_cash_account
+
+	expense_claim = frappe.get_doc("Expense Claim", docname)
+	default_bank_cash_account = get_default_bank_cash_account(expense_claim.company, "Bank")
+
+	je = frappe.new_doc("Journal Entry")
```

Comment:
@rmehta As of now, Payment Entry cannot be made against Expense Claim, will implement later.